Bumping versions to 0.33.3 (#302)
square/Blueprint: CHANGELOG.md

# Past Releases

## [0.33.3] - 2021-12-8

### Fixed

- Fixed an issue where `Box` did not implicitly animate its shadow.

### Changed

> - Reverted scroll view keyboard inset behavior to the behavior in `0.30.0`, as it was causing other issues.

Author: ```suggestion
- Reverted scroll view keyboard inset behavior to the behavior in `0.30.0`, since the recent changes were causing unexpected issues.
```
